moped-editor/src/queries/project.js

```diff
@@ -1021,3 +1021,12 @@ export const GET_PROJECTS_GEOGRAPHIES = gql`
     }
   }
 `;
+
+export const GET_PROJECT_SUBSTANTIAL_COMPLETION_DATE = gql`
+  query MopedProjectCompletionDate($projectId: Int) {
+    project_list_view(where: { project_id: { _eq: $projectId } }) {
+      substantial_completion_date
+      project_id
```

Comment:
I needed to include the project_id because of the inMemoryCache keyFields settings